sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -1,11 +1,47 @@
 namespace Azure.Data.Tables
 {
+    public partial class DynamicTableEntity : System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<string, object>>, System.Collections.Generic.IDictionary<string, object>, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object>>, System.Collections.IEnumerable
```

Comment:
Why doesn't this derive from `TableEntity`?

Reply:
Since all the properties and constructor would be overridden, would it better to have an `ITableEntity` interface? Track 1 does this!

Reply:
[We try to use classes whenever we can](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/interface) and only resort to interfaces when we don't have other options.  I'd start with a class if we can make that work.